src/components/RouteWrapper.tsx

```diff
@@ -76,7 +76,7 @@ import WhatWeb from "./WhatWeb/WhatWeb";
 import Sublist3r from "./Sublist3r/Sublist3r";
 import Arpscan from "./ArpScan/ArpScan";
 import Whois from "./Whois/Whois";
-import Masscan from "./Masscan/Masscan";
+import TestSSL from "./Testssl/Testssl";
```

Comment:
Probably best not to delete Masscan. ;-)
```suggestion
import Masscan from "./Masscan/Masscan";
import TestSSL from "./Testssl/Testssl";
```

Reply:
Sorry about that! I was working on my own masscan before I realized it was already being developed by another person on the project